workers/generic-worker/schemas/multiuser_posix.yml

```diff
@@ -297,6 +297,18 @@ oneOf:
             `exception/malformed-payload`.
 
             Since: generic-worker 54.5.0
+        runTaskAsCurrentUser:
+          type: boolean
+          title: Run task as current user
+          description: |-
+            If true, users will still be created for tasks, but
+            tasks will be executed as the current OS user.
```

Comment:
```suggestion
            If `true`, task commands will be executed as the
            user currently running Generic Worker (typically
            `root` or `LocalSystem`), rather than as the
            dedicated task user created for the task. The task
            user account will still be created, and is
            available for the task to use.
```